### PR TITLE
docs(accordion): add api docs

### PR DESCRIPTION
**Pull request by wesleycho**
- Add api docs for the accordion and panel components

### CODE REVIEW

File: src/accordion/accordion.ts [hunk 1]
```diff
@@ -33,9 +33,25 @@ let nextId = 0;
   directives: [NgbCollapse]
 })
 export class NgbPanel {
+  /**
+   *  A flag determining whether the panel is disabled or not.
+   *  When disabled, the panel cannot be toggled.
```
Comment (pkozlowski-opensource) [inline src/accordion/accordion.ts]: cannot be toggled -> cannot be selected?

File: src/accordion/accordion.ts [hunk 2]
```diff
@@ -33,9 +33,25 @@ let nextId = 0;
...
+   */
   @Input() disabled = false;
+
+  /**
+   *  An optional id for the panel. The id should be unique.
```
Comment (pkozlowski-opensource) [inline src/accordion/accordion.ts]: I would add a note: "If not provided by a user it will be auto-generated." or sth along those lines.

**Comment by pkozlowski-opensource** [comment]
LGTM, just left 2 remarks.

File: src/accordion/accordion.ts (hunk 1 above)
Comment (wesleycho) [inline src/accordion/accordion.ts]: If it is open and later becomes disabled, then one will not be able to close the panel. This language was intentional.

Comment (pkozlowski-opensource) [inline src/accordion/accordion.ts]: You are right. Don't know why but I had tabset on my mind while I was writing this....

**Comment by wesleycho** [comment]
Merging this as per comments

**Pull request closed without merging**